### ISSUE TITLE
New release containing Django 2.2 support?

### ISSUE THREAD

**Issue by stevejalim** [justinmayer/django-elevate#11]
Hi folks

I can see that master contains a fix for a regression under Django 2.1.x and above

```
  File "/usr/local/lib/python3.7/site-packages/elevate/views.py", line 64, in dispatch
    if not is_safe_url(url=redirect_to, host=request.get_host()):
TypeError: is_safe_url() got an unexpected keyword argument 'host'
```

Your fix: https://github.com/justinmayer/django-elevate/commit/f3ca619de2ca51c5754df8497a2e4ac3e098a67b

I can pin pip to a the relevant commit for now, but am wondering if a 1.0.1 release is on the way, please. I appreciate everyone has a lot on their plate :o)

Cheers
Steve

**Comment by moggers87** [comment]
As Django 2.2 has just been released, I think we should get support for that in first.

**Comment by stevejalim** [comment]
2.2 works for me 👍

**Comment by stevejalim** [comment]
Thanks for the work supporting Django 2.2 :heart: 

Does anyone with the appropriate permissions have time to push a new release to pypi, please?

**Comment by justinmayer** [comment]
Thank you for the kind words and thoughtfully-expressed reminder. Will endeavor to get a release issued as soon as possible.

**Comment by justinmayer** [comment]
I just released Elevate 1.0.1, incorporating support for Django 2.1 and 2.2. Many thanks and a big shout-out to @moggers87 for making that happen! 🥇💯
